### ISSUE TITLE
Pangeo showcase talk for PyGMT

### ISSUE THREAD

**Issue by meghanrjones** [GenericMappingTools/pygmt#1462]
Pangeo recently posted a call for 15 minute virtual presentations during the (northern hemisphere) Fall 2021 Pangeo Showcase Seminar Series: https://twitter.com/pangeo_data/status/1430160570146242570?s=20.

I think this could be a good opportunity to spread the word about PyGMT even more. I would be glad to either lead the talk or support someone else's presentation about PyGMT.

The first step would be filling out a super simple application and then waiting to hear if the organizers are interested. Any opinions?

**Comment by willschlitzer** [comment]
Should we close this issue?

**Comment by weiji14** [comment]
Let's see if they continue the seminar series next year. I think it'll be good to connect with the Pangeo folks at some point.

**Comment by meghanrjones** [comment]
I agree that it still could be a fruitful avenue in the future. It might be nice to time a showcase talk with a first paper (xref #677).

**Comment by yvonnefroehlich** [comment]
Hey Max, thanks for asking! I don’t think I have time to lead such a presentation in the next months. But if my time allows, I would be happy to help with discussing ideas and reviewing materials or slides. However, I have to point out that I am currently not part of the Pangeo community and have to check out what is excepted for such a contribution.

**Comment by maxrjones** [comment]
> Hey Max, thanks for asking! I don’t think I have time to lead such a presentation in the next months. But if my time allows, I would be happy to help with discussing ideas and reviewing materials or slides. However, I have to point out that I am currently not part of the Pangeo community and have to check out what is excepted for such a contribution.

oh good point, I should have specified the expectations in my comment. A showcase talk is a 15-30 minute presentation on Zoom followed by a discussion period lasting up to 20 minutes. The talks are recorded and uploaded to YouTube/Zenodo and assigned a DOI. You can take a look at past showcase presentations through the [Discourse announcements](https://discourse.pangeo.io/c/meta/pangeo-showcase/19). The audience is usually technically oriented software developers and scientists. During the spring and fall, Pangeo hosts a showcase presentation each Wednesday. This spring, we're trying out the last Wednesday of the month being during an Europe-friendly time zone. Most of the spring schedule is still open.

**Comment by weiji14** [comment]
Hi Max, could we bookmark this for the second half of 2025 (during the Northern Hemisphere Autumn?). Specifically, I'd be keen to do a showcase talk once we get the PyArrow integration (#2800) and some of the internal tempfile to virtualfile refactorings (e.g. #3468) done, that might require at least 2 more releases (so after PyGMT v0.16.0 or v0.17.0).

**Comment by maxrjones** [comment]
> Hi Max, could we bookmark this for the second half of 2025 (during the Northern Hemisphere Autumn?). Specifically, I'd be keen to do a showcase talk once we get the PyArrow integration ([#2800](https://github.com/GenericMappingTools/pygmt/issues/2800)) and some of the internal tempfile to virtualfile refactorings (e.g. [#3468](https://github.com/GenericMappingTools/pygmt/pull/3468)) done, that might require at least 2 more releases (so after PyGMT v0.16.0 or v0.17.0).

Sounds good, I'll reach out when it's time to schedule the fall series. Thanks for your consideration!
